src/site/content/en/blog/overloaded-server/index.md

```diff
@@ -0,0 +1,241 @@
+- Click the 'Size' column to sort the image files by size
+
+#### Fix
+
+*If you have limited time…*
+
+Focus your time on Identifying large and frequently loaded images and manually optimizing them with a tool like [Squoosh](https://squoosh.app/). Hero images are often good candidates for optimization.
+
+Things to keep in mind:
+*   Size: Images should be no larger than necessary.
+*   Compression: Generally speaking, a quality level of 80-85 will have a minimal effect on image quality while yielding a 30-40% reduction in file size.
+*   Format: Use JPEGs for photos rather than PNG; use MP4 for [animated content](https://web.dev/replace-gifs-with-videos/) rather than GIF.
+
+*If you have more time…*
+
+Consider setting up an image CDN if images make up a non-negligible portion of your site. Image CDNs are designed for serving and optimizing images and they will offload image serving from the origin server. Setting up an image CDN is straightforward but requires updating existing image URLs to point at the image CDN.
+
+Further reading:
+*   [Use image CDNs to optimize images](https://web.dev/image-cdns/#optimize-your-images)
+*   [images.guide](https://images.guide/)
+
+
+### Minify JS and CSS
+
+Minification removes unnecessary characters from JavaScript and CSS.
+
+#### Diagnose
+
+Use the [Minify CSS](https://developers.google.com/web/tools/lighthouse/audits/minify-css) and [Minify JavaScript](https://web.dev/unminified-javascript/) Lighthouse audits to identify resources that are in need of minification.
+
+#### Fix
+
+If you have limited time, focus on minifying your JavaScript. Most sites have more JavaScript than CSS, so this will be more impactful.
+*   [Minify JavaScript](https://web.dev/reduce-network-payloads-using-text-compression/)
+*   [Minify CSS](https://web.dev/minify-css/)
+
+
+## Monitor
+
+Server monitoring tools provide data collection, dashboards, and alerting regarding server performance. Their usage can help prevent and mitigate future server performance issues.
+
+A monitoring setup should be kept as simple as possible. Excessive data collection and alerting has its costs: the greater the scope or frequency of data collection, the more expensive it is to collect and store; excessive alerting inevitably leads to ignored pages.
+
+Alerting should use metrics that consistently and accurately detect issues. Server response time (latency) is a metric that works particularly well for this: it catches a wide variety of issues and correlates directly with user experience. Alerting based on lower-level metrics like CPU usage can be a useful supplement but will catch a smaller subset of issues. In addition, alerting should be based on the performance observed at the tail (i.e. 95th or 99th percentiles), rather than averages. Otherwise, averages can easily obscure issues that don't affect all users.
```

Comment:
```suggestion
Alerting should use metrics that consistently and accurately detect issues. Server response time (latency) is a metric that works particularly well for this: it catches a wide variety of issues and correlates directly with user experience. Alerting based on lower-level metrics like CPU usage can be a useful supplement but will catch a smaller subset of issues. In addition, alerting should be based on the performance observed at the tail (in the other words the 95th or 99th percentiles), rather than averages. Otherwise, averages can easily obscure issues that don't affect all users.
```